lib/pillar.ex

```diff
@@ -20,7 +20,10 @@ defmodule Pillar do
   end
 
   def query(%Connection{} = connection, query, params \\ %{}, options \\ %{}) do
-    final_sql = QueryBuilder.query(query, params)
+    final_sql =
+      QueryBuilder.query(query, params)
+      |> IO.inspect()
```

Comment:
in Final PR lets remove all IO.inspect

Reply:
They can be replaced by Logger.info

Reply:
Yeah will do so. I just deployed current dev state so you could take a look already. Will try to get more things rolled out and add some tests. Hope to find some time soon! :)